Automattic/wp-calypso
# Style Campaigns filter for BlazePress

Related to https://github.com/Automattic/wp-calypso/pull/78076#issuecomment-1588749011

## Proposed Changes
* Style Campaigns filter according to new [design](https://www.figma.com/file/uvh2KtY65b7hGBHI9exbJ1/BlazePress?type=design&node-id=5448-44706&t=06QUGdB1wg5Ey4Rt-0) (both mobile and desktop).
* Add missed margin-top for mobiles when campaigns not found
<img width="512" alt="Screenshot 2023-06-13 at 16 35 39" src="https://github.com/Automattic/wp-calypso/assets/115007291/ef2e27f1-46a7-4b2e-a751-bfe7887c237d">

## Testing Instructions
- Checkout this branch to your development environment
- Build assets (`yarn && yarn start`)
- Try campaign filter on **mobile**-sized screens like that:
![switch-campaign-filter-mobile git](https://github.com/Automattic/wp-calypso/assets/115007291/63b58d84-af46-4638-b9a7-07926dc7ab55)

- Try the same but on desktop-ish screen
![switch-campaign-filter-desktop git](https://github.com/Automattic/wp-calypso/assets/115007291/6837e6cf-2e8c-4952-a25b-e997bd80ee8b)

- Now make sure that mobile view have 24px margin when campaigns not found (to do so you need some campaign and using a filter that returns nothing; if you don't have campaigns at all, it won't work). You should ensure that there's a gap before a filter and a sign:
<img width="512" alt="Screenshot 2023-06-13 at 16 35 39" src="https://github.com/Automattic/wp-calypso/assets/115007291/ef2e27f1-46a7-4b2e-a751-bfe7887c237d">

- And the same for desktop:
<img width="937" alt="image" src="https://github.com/Automattic/wp-calypso/assets/115007291/625ff25d-e336-46e5-abec-3f658c9fd537">




## Pre-merge Checklist
